workshops.md

```diff
@@ -0,0 +1,333 @@
+Help us make one!  We're looking to bring together data experts with
+teaching interests together with teaching experts with data interests
+for a week of collaborative conversion of these materials into
+smoothed-out tutorials for use in undergraduate classrooms. Depending
+on the number of people attending and the interest they bring, we will
+work on one or more of the following topic areas: Genome assembly,
+RNAseq analysis and/or 16s rRNA microbial community analysis.
+ 
+Attendees should have some familiarity with the NGS workshop materials
+(perfect for those who have just taken it!) and attendees with
+professional expertise in the topic areas are particularly welcome. We
+will provide basic training in the use of GitHub for collaborative
+work. Funding options and strategies to broaden opportunities for
+bioinformatics in undergraduate settings will be discussed.
+
+#### [Workshop registration](https://registration.genomecenter.ucdavis.edu/events/DIBSI_undergraduate_curriculum_hackathon/)
+
+#### [Housing registration](https://registration.genomecenter.ucdavis.edu/events/Housing/)
+
+### Introduction to Python
+
+Dates: July 10-15
+
+Instructor: [Emily Dolson](mailto:EmilyLDolson@gmail.com)
+
+This workshop will introduce students to the general-purpose
+programming language, Python. Attendees will be researchers with
+problems that could be solved with programming, such as simple
+automated text-mining tasks, visualization of complex data, or
+pipeline scripting across a large-scale data set. As time permits, the
+Python scientific ecosystem (pandas, numpy, scipy, seaborn,
+matplotlib, etc) will be introduced to get learners up to speed on the
+ins and outs of using the tools that are currently most popular.
+
+Before the workshop begins, each learner may identify a problem that
+they would like to be able to solve with programming and run it by the
+instructor: Emily Dolson (Michigan State University),
+EmilyLDolson@gmail.com, who will then focus the workshop around
+teaching the appropriate skills and coming up with challenge problems
+to meet the needs of the attendees.
+
+#### [Workshop registration](https://registration.genomecenter.ucdavis.edu/events/DIBSI_Intro_python_workshop/)
+
+#### [Housing registration](https://registration.genomecenter.ucdavis.edu/events/Housing/)
+
+### Reproducible research with R/Data Hackathon
+
+Dates: July 10-15
+
+Instructors: [Chris Hamm](mailto:cahamm@ucdavis.edu)
+
+This workshop will be a hybrid between a two-day Software Carpentry
+style workshop followed by a hack-a-thon. In this workshop
+participants will learn the fundamentals of employing reproducible
+research in the R language, followed by three days of instructor-led
+group working sessions where participants will employ the skills just
+acquired towards their own research projects.
+
+Target audience: everyone is welcome: grad students, post-docs,
+faculty, everyone. Idealy, particpants will have a working knowledge
```

Comment:
I was double checking the content crossover with the other R course and saw 2 small typos. Just a heads up!

*Ideally
*participants